TodoBasicWithAuth/CreateUser.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Todos
+{
+    public class CreateUser
+    {
+        [Required]
+        public string UserName { get; set; }
+
+        [Required]
+        public string Password { get; set; }
+
+        public bool IsAdmin { get; set; }
```

Comment:
I'm not comfortable with this. I think the IsAdmin logic should stay server side. Even though this is a demo app 😄

Reply:
Ah no worries, I could revert the CreateUser back to LoginInfo so users can only be created as a regular user via the API then seed the Admin user on startup? Or is that overkill for this project?

Reply:
Either that or the first user is admin with a comment describing that.

Reply:
Reverted the insecure user creation back to the original implementation and seeded admin on startup. sample.http now works with the new approach